viz/charts/linkerd-viz/templates/prometheus.yaml

```diff
@@ -245,6 +245,9 @@ spec:
         {{- if not (hasKey .Values.prometheus.args "log.level") }}
         - --log.level={{.Values.prometheus.logLevel | default .Values.defaultLogLevel}}
         {{- end }}
+        {{- if not (hasKey .Values.prometheus.args "log.format") }}
+        - --log.format={{.Values.prometheus.logFormat | default "logfmt" }}
```

Comment:
Same here, we should transform the values from `.Values.defaultLogFormat` to the values accepted by prometheus.

Reply:
Sorry for the delay! I'm sending the changes right away. I thought of putting a simple replace at the end of the chain, so, in the event that someone wants to configure prometheus / grafana to another log format, they can still do it.

Reply:
I could also mimic the behavior for linkerd-components (i.e. if json set to json, else set to plain / logfmt). This would have the benefit to really only allow these two options.